### ISSUE TITLE
Handle post type in single.twig

### ISSUE THREAD

**Issue by wroxbox** [forumone/gesso-theme-wordpress#98]
In wp people do not use post-format (audio,gallery, image,video) etc types so much any more. Instead having support for post types in `single.twig`

instead of:
` {% include 'post-formats/content-' ~ function( 'get_post_format' ) ~ '.twig' %}`

I created 
`
{% if function( 'get_post_type' ) %}
          {% include 'post-types/content-' ~ function( 'get_post_type' ) ~ '.twig' %}
{% endif %}
`

and created for each post type a separate content-myposttype.twig file based on post-format/content.twig -file.

What do you think?

Of course the same can be done copying the single.twig to have single-myposttype.twig.

**Comment by cssgirl** [comment]
Looking into it this week @coreylafferty

**Comment by cssgirl** [comment]
I agree with this. We don't typically utilize post formats in most builds. 

I implemented this, with a few minor changes.

Updated conditional in single.twig to point to the new `content` folder. It will look for a file that matches the current post type prepended with `content-`. If none exists it uses the default `content.twig`.

```
{% if fn( 'get_post_type' ) %}
    {% include 'content/content-' ~ fn( 'get_post_type' ) ~ '.twig' %}
{% else %}
    {% include 'content/content.twig' %}
{% endif %}
```

I updated the `post-formats` folder name to `content`. I removed the files related to post formats except for `content.twig`.

When new post types are added their specific files should be named `content-post-type-name`. Ex: `content-resource.twig`.

I also wonder if we should change the naming scheme we use most often for blocks. Currently when we create new blocks, their files get saved to the `blocks` folder typically named `content-block-name` - should we recommend naming be changed to be more related to blocks like `block-block-name`?

Work in this PR: https://github.com/forumone/gesso-wp/pull/208
